src/App/Material.cpp

```diff
@@ -25,10 +25,26 @@
 
 #ifndef _PreComp_
 #include <cstring>
+#include <random>
 #endif
 
+#include "Application.h"
 #include "Material.h"
 
+// Helper functions to consistently convert between float and long
+namespace
+{
+float fromPercent(long value)
+{
+    return static_cast<float>(value) / 100.0F;
+}
+
+long toPercent(float value)
+{
+    return static_cast<long>(100.0 * value + 0.5);

```

Comment:
The linter is (almost) right here, this code should use `std::roundl()`.

Reply:
This is plagiarized code. I'll update.